tests/test_hbg_dataset.py

```diff
@@ -62,24 +64,24 @@ def test_main_dat(ontology, NWB_file):
     compare_dicts(expected, sweep_record)
 
 
-@pytest.mark.parametrize('ontology, NWB_file', [(None, 'H18.28.015.11.14.nwb')], indirect=True)
+@pytest.mark.parametrize('ontology, NWB_file', [(None, 'H20.28.008.11.05-10.nwb')], indirect=True)
 def test_get_clamp_mode(ontology, NWB_file):
 
-    dataset = HBGDataSet(nwb_file=NWB_file, ontology=ontology)
+    dataset = HBGNWB2Data(nwb_file=NWB_file, ontology=ontology)
     assert dataset.get_clamp_mode(10101) == dataset.VOLTAGE_CLAMP
 
 
-@pytest.mark.parametrize('ontology, NWB_file', [(None, 'H18.28.015.11.14.nwb')], indirect=True)
```

Comment:
why not use the original file? it should be nwb version 2.

Reply:
modified the test_hbg_data.py file avoid using the original file